### PR TITLE
Add cafe menu

### PR DESCRIPTION
**Pull request by Gbemisola19**
## Cafe Menu
## Built with:
* Html elements
`div` `section`
* CSS 
`font-size`, `margin`, `color`, `text-align`, `display`

### CODE REVIEW

**Comment by Gbemisola19** [comment]
Added a **cafe menu** with different classes of drinks using html and CSS

**Comment by Gbemisola19** [comment]
Added a **Cafe menu** with classes of drinks using html and CSS

**Comment by Gbemisola19** [comment]
This project shows information about a **Cafe Menu**

File: README.md [hunk 1]
```diff
@@ -1,27 +1,36 @@
...
 
 Additional description of the project and its features.
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- HTML5 Elements,
+<div>, <select>
```
Comment (OMOTAYO05) [inline README.md]: Use code for the div and select

File: README.md [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
+- CSS,
+  selector {
```
Comment (OMOTAYO05) [inline README.md]: use code for the CSS selector

**Pull request merged**